src/ol_concourse/pipelines/infrastructure/consul/packer_pulumi_pipeline.py

```diff
@@ -22,12 +22,20 @@
         *PACKER_WATCHED_PATHS,
     ],
 )
-consul_pulumi_code = git_repo(
+consul_pulumi_infrastructure_code = git_repo(
     name=Identifier("ol-infrastructure-pulumi"),
     uri="https://github.com/mitodl/ol-infrastructure",
     paths=[
         *PULUMI_WATCHED_PATHS,
         "src/ol_infrastructure/infrastructure/consul/",
+    ],
+)
+
+consul_pulumi_substructure_code = git_repo(
+    name=Identifier("ol-infrastructure-pulumi"),
```

Comment:
Will it not complain about having the same `Identifier`?

Reply:
It did :) It should be fixed now.